carbonmark/lib/actions.ts

```diff
@@ -317,7 +317,7 @@ export const addProjectsToAssets = async (params: {
     });
     const projectIds = Array.from(projectIdSet);
     const projects = await Promise.all(
-      projectIds.map((id) => getCarbonmarkProject(id))
+      projectIds.map((id) => getProjectsId(id))
```

Comment:
Shouldn't this ideally be called `getProjectById`, leaving out the plural and adding `by`?

Reply:
Same for purchaseById etc.